### ISSUE TITLE
DF Active Lights: Conflict with Lock View (only when both mods enabled)

### ISSUE THREAD

**Issue by Shuggaloaf** [flamewave000/dragonflagon-fvtt#188]
**Module**
DF Active Lights 1.1.2
Lock View 1.4.12

**Describe the issue**
Reporting this issue to both module's devs as it only occurs when _both_ Lock View AND DF Active Lights are enabled. 
Sometimes when moving token to edge of screen, when map move transition would normally occur there is first a shaking/vibrating that happens before the map repositions. I tried to get a screencap for you both but it happens so fast it's no being picked up on the cap.
Odd thing is this was happening on a map with no lights setup yet and Lock View not turned on at all.
I determined it was only when both mods, not 1 or the other, were active by running numerous tests with Find the Culprit and then by testing on a fresh world with only those 2 mods (and Lib Wrapper) active.

Unfortunately no error messages are generated in console and no screen caps. Wish I could give you guys more to go on.

**To Reproduce**
Try having both mods active on a large maps with a medium+ amount of walls. As mentioned I had both mods enabled but no lights and no active lock view features. 
Zoom in very close on a token and walk the token to the edge of the screen. 
Should see vibration/shaking before the map moves. 

Link to other mod Issue Report:
https://github.com/CDeenen/LockView/issues/58

**Comment by Shuggaloaf** [comment]
Hi Flamewave. I had deactivated DF AL but I just re-enabled and the issue persists. the shaking is sort of a jitter/jump before token movement. I wish I could screencap this for you but am unable to. 

Did you catch the part where this only happens if the Lock View mod is enabled as well? Assuming you did, but just didn't see you mention that. If you had that enabled then you should see the issue (not specifically using it on the scene, just needs to be active). 

If I can find a way to get a higher fps screencap I will post back for you. Thanks for checking into this as I'd really like to be able to use DF AL.

**Comment by Shuggaloaf** [comment]
>hopefully I can find a fix for you

Thanks Flamewave. I know it's probably a more obscure mod combination so I understand it's not a top priority. I appreciate you digging into it though. 

Of course we _could_ just flip this whole thing into a request for a new LockView-like feature in DF Scene Enhancements to lock the position and zoom of a scene. 😉
(Mainly joking there but who knows, depending on how difficult the conflict is to resolve, that may be the easier way. lol)

**Comment by Shuggaloaf** [comment]
Hey Flamewave. Just had a reminder of this mod so came back to check on the status here. 

Since you have had a few updates and we're on a whole new system version, I decided to re-install the mod and see if the problem persists. As far as I can tell so far, the problem seems to have been fixed. Not sure if it was one of your updates or an outside factor but for now all seems to be well. 

I'm going to go ahead and close this and if I see it pop up again will re-open. Thanks!
